ext/strings.go

```diff
@@ -44,6 +57,45 @@ import (
 //	'hello'.charAt(5)  // return ''
 //	'hello'.charAt(-1) // error
 //
+// # Format
+//
+// Returns a new string with substitutions being performed, printf-style.
+// The valid formatting clauses are:
+//
+// `%s` - substitutes a string. This can also be used on lists, maps, bytes, Duration and Timestamp.
+//	Note that the dot/period decimal separator will always be used when printing a list or map that contains a double.
+// `%d` - substitutes an integer.
+// `%f` - substitutes a double with fixed-point precision. The default precision is 6, but
+// this can be adjusted. The strings `Infinity`, `-Infinity`, and `NaN` are also valid input
+// for this clause.
+// `%e` - substitutes a double in scientific notation. The default precision is 6, but this
+// can be adjusted.
+// `%b` - substitutes an integer with its equivalent binary string.
+// `%x` - substitutes an integer with its equivalent in hexadecimal, or if given a string or
+// bytes, will output each character's equivalent in hexadecimal.
+// `%X` - same as above, but with A-F capitalized.
+// `%o` - substitutes an integer with its equivalent in octal.
+// `%t` - substitutes a boolean with the strings "true" or "false".
```

Comment:
I think I mentioned the `%t` and `%T` could collide with Java `String.format` formatting clauses for time. Wouldn't the boolean case here be covered by our definition of `%s` and `%b`?

Reply:
(marking as resolved now that `%s` and `%b` both handle bools)